src/main/java/org/kapott/hbci/tools/ParameterFinder.java

```diff
@@ -48,6 +48,16 @@ public static class Query
          */
         public final static Query BPD_PINTAN_ORDERHASHMODE = new Query("Params_*.TAN2StepPar{0}.ParTAN2Step*.orderhashmode",true);
 
+        /**
+         * Die 3 verschiedenen Parameter für status refresh requests beim Decoupled Verfahren:
+         *  1. Minimale Zeit vor dem ersten refresh (Sekunden).
+         *  2. Minimale Zeit vor weiteren refreshes (Sekunden).
+         *  3. Maximale Anzahl von refreshes.
+         */
+        public final static Query BPD_DECOUPLED_TIME_BEFORE_FIRST_STATUS_REQUEST = new Query("Params_*.TAN2StepPar7.ParTAN2Step*.TAN2StepParams_*.decoupled_time_before_first_status_request",false);
```

Comment:
Kannst du bei den 3 Queries vielleicht noch die Versionsnummer "7" beim Segment "TAN2StepPar7" entfernen? Entweder per ""TAN2StepPar*" oder per "TAN2StepPar{0}" (siehe BPD_PINTAN_ORDERHASHMODE - dann muss aber beim Query die Version übergeben werden.

Grund: Das funktioniert sonst nur bei HKTAN in Version 7. Derzeit ist das der Fall. Wenn aber irgendwann mal eine Version 8 von HKTAN eingeführt wird, würden die Queries ins Leere laufen.